code/modules/mob/living/basic/vermin/stickman.dm

```diff
@@ -0,0 +1,85 @@
+/mob/living/basic/stickman
+	name = "Angry Stick Man"
+	desc = "A being from a realm with only 2 dimensions. At least it's trying to stay faced towards you."
+	icon_state = "stickman"
+	icon_living = "stickman"
+	icon_dead = "stickman_dead"
+	mob_biotypes = MOB_HUMANOID
+	gender = MALE
+	health = 100
+	maxHealth = 100
+	speed = 0
+	attack_verb_continuous = "punches"
+	attack_verb_simple = "punch"
+	melee_damage_lower = 10
+	melee_damage_upper = 10
+	attack_sound = 'sound/weapons/punch1.ogg'
+	combat_mode = TRUE
+	faction = list("stickman")
+
+	ai_controller = /datum/ai_controller/basic_controller/stickman
+
+/mob/living/basic/stickman/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/basic_body_temp_sensitive)
+	AddElement(/datum/element/atmos_requirements, list("min_oxy" = 5, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 1, "min_co2" = 0, "max_co2" = 5, "min_n2" = 0, "max_n2" = 0), 7.5)
+
+/datum/ai_controller/basic_controller/stickman
+	blackboard = list(
+		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic()
+	)
+
+	ai_movement = /datum/ai_movement/basic_avoidance
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/simple_find_target,
+		/datum/ai_planning_subtree/basic_melee_attack_subtree/stickman,
+		/datum/ai_planning_subtree/find_and_hunt_target
+	)
+
+/datum/ai_planning_subtree/basic_melee_attack_subtree/stickman
+	melee_attack_behavior = /datum/ai_behavior/basic_melee_attack/stickman
+
+/datum/ai_behavior/basic_melee_attack/stickman
+	action_cooldown = 1.5 SECONDS
+
+/datum/ai_controller/basic_controller/stickman/PerformIdleBehavior(delta_time)
```

Comment:
Not a note for you necesarily but I'll make idle behaviors into a datum. this piece of code is copy pasted all over the place.